cumulus/parachains/runtimes/testing/yet-another-parachain/src/lib.rs

```diff
@@ -304,6 +304,7 @@ impl cumulus_pallet_parachain_system::Config for Runtime {
 	type CheckAssociatedRelayNumber = RelayNumberMonotonicallyIncreases;
 	type ConsensusHook = ConsensusHook;
 	type SelectCore = cumulus_pallet_parachain_system::DefaultCoreSelector<Runtime>;
+	type RelayParentOffset = ConstU32<0>;
```

Comment:
let's try it :)
```suggestion
	type RelayParentOffset = ConstU32<1>;
```

Reply:
I guess it would imply bumping unincluded segment to 10?

Reply:
You also need to implement the runtime API to make this work.

Reply:
shouldn't be `RELAY_PARENT_OFFSET`?